<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultiTagValuesDialogClass</class>
 <widget class="QDialog" name="MultiTagValuesDialogClass">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>467</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ActionsContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Edit values</string>
  </property>
  <property name="windowIcon">
   <iconset resource="analogexif.qrc">
    <normaloff>:/images/icons/film_edit.png</normaloff>:/images/icons/film_edit.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Edit multiple values</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="addBtn">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="analogexif.qrc">
         <normaloff>:/images/icons/add.png</normaloff>:/images/icons/add.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="delBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="analogexif.qrc">
         <normaloff>:/images/icons/delete.png</normaloff>:/images/icons/delete.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="upBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="analogexif.qrc">
         <normaloff>:/images/icons/bullet_arrow_up.png</normaloff>:/images/icons/bullet_arrow_up.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="downBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="analogexif.qrc">
         <normaloff>:/images/icons/bullet_arrow_down.png</normaloff>:/images/icons/bullet_arrow_down.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="valuesView">
     <property name="font">
      <font>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>16</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>16</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionAdd_new_value">
   <property name="text">
    <string>Add new value</string>
   </property>
   <property name="shortcut">
    <string>Ins</string>
   </property>
  </action>
  <action name="actionDelete_value">
   <property name="text">
    <string>Delete value</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionMove_up">
   <property name="text">
    <string>Move up</string>
   </property>
  </action>
  <action name="actionMove_down">
   <property name="text">
    <string>Move down</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>valuesView</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>addBtn</tabstop>
  <tabstop>delBtn</tabstop>
  <tabstop>upBtn</tabstop>
  <tabstop>downBtn</tabstop>
 </tabstops>
 <resources>
  <include location="analogexif.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>addBtn</sender>
   <signal>clicked()</signal>
   <receiver>actionAdd_new_value</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delBtn</sender>
   <signal>clicked()</signal>
   <receiver>actionDelete_value</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>upBtn</sender>
   <signal>clicked()</signal>
   <receiver>actionMove_up</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>downBtn</sender>
   <signal>clicked()</signal>
   <receiver>actionMove_down</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
